backend/app/routers/files.py

```diff
@@ -43,6 +43,57 @@
 router = APIRouter()
 security = HTTPBearer()
 
+
+async def _resubmit_file_extractors(
+    file_id: str,
+    credentials: HTTPAuthorizationCredentials = Security(security),
+    db: MongoClient = Depends(dependencies.get_db),
+    rabbitmq_client: BlockingChannel = Depends(dependencies.get_rabbitmq),
+):
+    """This helper method will check metadata. We get the extractors run from metadata from extractors.
+    Then they are resubmitted. At present parameters are not stored. This will change once Jobs are
+    implemented.
+
+        Arguments:
+        file_id: Id of file
+        credentials: credentials of logged in user
+        db: MongoDB Client
+        rabbitmq_client: Rabbitmq Client
+
+    """
+    if (file := await db["files"].find_one({"_id": ObjectId(file_id)})) is not None:
```

Comment:
In the spot where you call this method, you already have the FileOut you could pass in directly and not have to query it again here.

Reply:
fixed that.